examples/ft_transformer_text.py

```diff
@@ -1,7 +1,9 @@
 import argparse
+import os
 import os.path as osp
 from typing import List
 
+import cohere
```

Comment:
Move this to the cohere embedding class? Now it requires user install cohere to run the text example.

Reply:
Makes sense